### ISSUE TITLE
Final Fantasy type-0 texture bug

### ISSUE THREAD

**Issue by andreyorst** [hrydgard/ppsspp#9153]
**Device:** Nexus 5X;
**OS:** Android 7.1.1;
**PPSSPP Version:** 1.3.0.1;

Some weird texture glitch appeared in game. There is nothing extraordinary in settings, everything pretty much default.
See what i mean:
![image](https://cloud.githubusercontent.com/assets/19470159/20922243/86604d56-bbb8-11e6-8cd4-10b005eec3b8.png)
![image](https://cloud.githubusercontent.com/assets/19470159/20922248/8915bad6-bbb8-11e6-8ef8-cb4646cc3ee8.png)
![image](https://cloud.githubusercontent.com/assets/19470159/20922258/93b4a2ae-bbb8-11e6-96ef-8aebd722f4c1.png)
Seems that textures from Queen (that girl at the left side of screen) were applied to Ace. Some times Ace becomes transparent, or "white" like without textures at all.

I've made log, using "save next frame to log" button in settings. here it is:
https://drive.google.com/drive/folders/0B4tzN5LjfRj1UllRWXg4ZXlXYUk?usp=sharing
But i don't know will it help in any way to figure out what happened

**Comment by zminhquanz** [comment]
Turn off multithreading may help , this texture is glitch when you enable multithreading , some game hate multithreading because it's experimental

**Comment by andreyorst** [comment]
@zminhquanz, sure it helps, but performance is decreased as i can feel.

**Comment by LunaMoo** [comment]
Buggy options exist in PPSSPP due to popular demand, nobody forces you to use them, but if you do, be wary of consequences. In any fairy new PPSSPP version there is a message informing you about problems when using certain buggy options like multithreading. Ignoring this warning is your choice. Note that to fully clear state of whatever hackish and broken settings you used, you might need to restart emulation and reload from in-game save since savestates will keep any memory corruption some buggy settings can cause.

 About MT - fixing it means more synchronization which means it will become slower, eventually slower than without which as a matter of fact already happens at least on some hardware/in some games and people who use it doesn't want that.
 In other words don't expect this to get fixed unless someone in the future finds a better way of separating gpu to it's own thread, that could work reasonably fast without breaking games.

**Comment by zminhquanz** [comment]
I hope developer to contribute new way about Multithreading in future

**Comment by andreyorst** [comment]
@LunaMoo thanks alot for your reply. I use ppsspp for quite long time (about 2 years or more) and as i can say - it gets better and better. I have read warnings of all buggy options, but i thought it was game specific. Thanks for your instruction about restarting emulation, didnt think that it might help.
